contracts/Vault.vy

```diff
@@ -299,6 +300,7 @@ def initialize(
     @param guardian The address authorized for guardian interactions. Defaults to caller.
     """
     assert self.activation == 0  # dev: no devops199
+    assert DetailedERC20(token).decimals() == 18
```

Comment:
this is not the case for all tokens tho? maybe what we really want to do is to force 18 decimal denom for vault shares.